pkg/sockops/sockops.go

```diff
@@ -60,7 +60,12 @@ const (
 	oIPC = "bpf_redir.o"
 	eIPC = "bpf_redir"
 
-	sockMap = "sock_ops_map"
+	sockMap = "cilium_sock_ops_map"
+
+	// Cilium version 1.3.0 used this map name, we remove it on
```

Comment:
I think this should be updated to say 1.4, and I don't think we need to backport this PR to 1.4 just because sockmap is still in some form of tech preview/beta.

Reply:
@joestringer Thanks, updated and rebased.